public/app/routes/routes.tsx

```diff
@@ -85,8 +85,10 @@ export function getAppRoutes(): RouteDescriptor[] {
       pageClass: 'dashboard-solo',
       routeName: DashboardRoutes.Normal,
       chromeless: true,
-      component: SafeDynamicImport(
-        () => import(/* webpackChunkName: "SoloPanelPage" */ '../features/dashboard/containers/SoloPanelPage')
+      component: SafeDynamicImport(() =>
+        config.featureToggles.dashboardScene || config.featureToggles.dashboardSceneSolo
```

Comment:
Just curious, what would be the case when only `dashboardSceneSolo` is enabled? It looks like as long as `dashboardScene` is enabled solo will work too

Reply:
I guess this should be config.featureToggles.dashboardScene && config.featureToggles.dashboardSceneSolo